packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatusWheel/ProgressWheel.tsx

```diff
@@ -176,23 +185,32 @@ export const ProgressWheel = ({ accountKey, togglePause }: ProgressWheelProps) =
         isSessionActive,
         togglePause,
         dispatch,
+        isCoinjoinUneco,
     ]);
 
+    const getTooltipMessage = () => {
+        if (isAccountEmpty) {
```

Comment:
This case is already included in `coinjoinSessionBlockedMessage` variants. The implementation in `coinjoinSessionBlockedMessage` is more robust as it takes into account UTXOs too small for coinjoin.

Reply:
Then i'd say the translation for that case should be changed. `Amounts are too small for coinjoin` implies you have _some_ amounts. While `Nothing to anonymize` is explicit in the fact that there is nothing on the account.

Reply:
Ok, you are right.